documentation/asciidoc/stories/assembly_eventing.adoc

```diff
@@ -14,6 +14,12 @@ Sending cloud events with {openshift} Serverless is available as a technology pr
 ====
 
 include::{topics}/ref_tech_preview.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+This feature is deprecated and planned for removal in a future release.
```

Comment:
I think we need to be stronger. We should say something like:

"This feature is deprecated and has no effect when utilising Infinispan 15 or higher."

Reply:
Thanks, @ryanemerson, updated!